### DISCUSSION TITLE
Push floating panel above tab bar

### DISCUSSION THREAD

**Discussion by veloxls200** [scenee/FloatingPanel#532] [Q&A] [answered]
Hello, I wanted to put the floating panel above the tab bar, so that we can access the tab bar items along with the floating panel.
Would really appreciate it if someone helps me with this, the desired layout is attached, I achieved this in android easily but don't know how to replicate the same on iOS swift
![Screenshot_2022-02-25-18-55-23-12_ae13db1dc042ce51650e77e4e36dc693__01](https://user-images.githubusercontent.com/33398794/155722854-2ea7beaa-380f-4d33-b2d3-e69698e966d9.jpeg)
.

**Comment by kaz080** [comment, marked as answer]
Maybe you added a floating panel as modality using `present(fpc, ...)`.

Instead, use `fpc.addPanel(toParent: self)` on a VC (in the tab bar controller) to be a child.
https://github.com/scenee/FloatingPanel#add-a-floating-panel-as-a-child-view-controller

**Comment by popeyelau** [comment]
👏
